### DISCUSSION TITLE
HLBFGS optimizer

### DISCUSSION THREAD

**Discussion by Gowrimv** [BrunoLevy/geogram#144] [Q&A] [answered]
Is optimization for CVT currently not included in geogram?

**Comment by BrunoLevy** [comment, marked as answer]
Yes it is,
it is implemented by the following components:
- [high level API](https://brunolevy.github.io/geogram/mesh__remesh_8h.html)
- [low level API](https://brunolevy.github.io/geogram/classGEO_1_1CentroidalVoronoiTesselation.html#details)
- [LBFGS](https://brunolevy.github.io/geogram/optimizer_8h.html)
